README.md

```
@@ -1,10 +1,65 @@
 # GourmetClient
 Vereinfachte Essenbestellung für Gourmet
 
-## Download
+## Download (Endanwender)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)